lib/tilex/notifications/notifiers/twitter.ex

```diff
@@ -1,7 +1,9 @@
 defmodule Tilex.Notifications.Notifiers.Twitter do
+  alias OAuther
   alias Tilex.Blog.Developer
 
   use Tilex.Notifications.Notifier
+  require Logger
```

Comment:
what's this used for?

Reply:
Oops that was for some debugging i forgot to remove it.